packages/game-app/cypress/integration/signup.spec.ts

```diff
@@ -0,0 +1,63 @@
+/// <reference types="Cypress" />
+/// <reference types="../support" />
+import "firebase/auth";
+
+context("Signup", () => {
+
+  const usedEmails: string[] = [];
+
+  beforeEach(() => {
+    cy.visit(Cypress.config().baseUrl!);
+  });
+
+  // TODO test for all required fields
+  it("should show invalid email error message", () => {
+    cy.getInputByName('email').type('non-valid-email');
+    cy.get('button').click();
+    cy.get('body').should('contain', 'signup.invalidEmail')
+  });
+
+  it("should show invalid password error", () => {
+    cy.getInputByName('password').type('123456789');
+    cy.get('button').click();
+    cy.get('body').should('contain', 'signup.passwordRequirements')
+  });
+
+  it("should show not contains invalid password error", () => {
```

Comment:
I would rephrase this with "should not show invalid password error for correct password"